src/oryx/install.sh

```diff
@@ -236,16 +234,19 @@ fi
 if [[ "${PINNED_SDK_VERSION}" != "" ]]; then
     rm -f ${GIT_ORYX}/global.json
     rm -rf /usr/share/dotnet/sdk/$PINNED_SDK_VERSION
-
-    # Extract the major, minor version and the first digit of the patch version
-    MAJOR_MINOR_PATCH1_VERSION=${PINNED_SDK_VERSION%??}
-    rm -rf /usr/share/dotnet/shared/Microsoft.NETCore.App/$MAJOR_MINOR_PATCH1_VERSION
-    rm -rf /usr/share/dotnet/shared/Microsoft.AspNetCore.App/$MAJOR_MINOR_PATCH1_VERSION
-    rm -rf /usr/share/dotnet/templates/$MAJOR_MINOR_PATCH1_VERSION
+    NEW_RUNTIME_VERSIONS=$(dotnet --list-runtimes | awk '{print $2}' | sort | uniq)
+    if [ -n "${RUNTIME_VERSIONS:-}" ]; then
+        SDK_INSTALLED_RUNTIME=$(echo "$NEW_RUNTIME_VERSIONS" | grep -vxFf <(echo "$RUNTIME_VERSIONS"))
+    else
+        SDK_INSTALLED_RUNTIME="$NEW_RUNTIME_VERSIONS"
```

Comment:
Are there cases where more than one sdk is installed? I see plural `NEW_RUNTIME_VERSIONS`
In that case `SDK_INSTALLED_RUNTIME` may get assigned a multiline string? Wouldn't that case issues with deletions on the lines below?

Reply:
No we install only one sdk as a part of oryx and this is installing only one runtime
It is in plural because for images with dotnet even before oryx they sometime might have multiple runtimes from those runtimes we pick only the runtime installed from oryx feature.

Reply:
In that case should the naming be updated to avoid the confusion?
I will go ahead and checkin this PR, you may followup with a PR if needed